mqlc/mqlc.go

```diff
@@ -121,6 +124,7 @@ func (c *compiler) newBlockCompiler(binding *variable) compiler {
 		blockRef:   ref,
 		props:      c.props,
 		standalone: true,
+		features:   c.features,
```

Comment:
I'd love to turn this into a compiler configuration in a follow-up, thank you for getting it started!